salt/metalk8s/repo/ubuntu.sls

```diff
@@ -0,0 +1,33 @@
+{%- from "metalk8s/map.jinja" import metalk8s with context %}
+{%- from "metalk8s/map.jinja" import repo with context %}
+
+{%- set repo_host = pillar.metalk8s.endpoints['repositories'].ip %}
+{%- set repo_port = pillar.metalk8s.endpoints['repositories'].ports.http %}
+
+Install python-apt:
+  pkg.installed:
+    - name: python-apt
+
+{%- for repo_name, repo_config in repo.repositories.items() %}
+Configure {{repo_name}} repository:
+  pkgrepo.managed:
+    - name : {{repo_config.name}}
```

Comment:
So you only handle local repositories, it seems that deploying on bootstrap node was the scope of the linked issue, but hey it does not cost much to handle http served repositories.